pyfftw/interfaces/__init__.py

```diff
@@ -128,6 +128,16 @@
 * :func:`pyfftw.interfaces.scipy_fftpack.rfft`
 * :func:`pyfftw.interfaces.scipy_fftpack.irfft`
 
+:mod:`~pyfftw.interfaces.numpy_fft`
+"""""""""""""""""""""""""""""""""""
+
+* :func:`pyfftw.interfaces.numpy_fft.fft`
+* :func:`pyfftw.interfaces.numpy_fft.ifft`
+* :func:`pyfftw.interfaces.numpy_fft.rfft`
+* :func:`pyfftw.interfaces.numpy_fft.irfft`
+* :func:`pyfftw.interfaces.numpy_fft.hfft`
+* :func:`pyfftw.interfaces.numpy_fft.ihfft`
```

Comment:
This should have been `dask_fft` instead of `numpy_fft`. PR ( https://github.com/pyFFTW/pyFFTW/pull/166 ) fixes this.